### ISSUE TITLE
Add SmartEdit to yenv spartacus template

### ISSUE THREAD

**Issue by Xymmer** [SAP/cloud-commerce-spartacus-storefront#368]
Acceptance criteria:

- Install an new yenv environment with the b2c_cms recipe.
- Access SmartEdit from the _legacy_ storefront to assess that SmartEdit is properly enabled/installed. (SmartEdit support in Spartacus is not yet implemented.)

**Comment by plabadie** [comment]
We could use the smartedit template directly, but there are unreleased modules that we use for convenience to eliminate manual configurations to get Spartacus working properly.

Result: The "b2c_cms" template not installs SmartEdit + Spartacus setup.

**Comment by hackergil** [comment]
I tested this today. Environment is built correctly. It can be started and I can browse smartedit. However, I found the following problems:
* There is no extended cart service in swagger
* When you try to checkout from spa, you get an infinite loop when requesting from the address form. Wei's suggestion for this was
` For a new user, there is no payments or addresses. Then in checkout step1 and step3, we will have this infinite loop issue.`
Both need to be addressed.
